src/components/ChapterDropDown.js

```diff
@@ -1,7 +1,34 @@
 import React from "react";
+import { IonSelect, IonSelectOption } from '@ionic/react';
+import PropTypes from 'prop-types';
 
-const ChapterDropDown = () => {
-  return <>Chapter DropDown</>;
+const ChapterDropDown = ({ navState, setNavState }) => {
+    
+    const setChapter = (e) => {
+        setNavState((prevState) => ({ ...prevState, chapter: e.detail.value }));
+    };
+
+  return (
+    <IonSelect
+        value={navState.chapter}
+        okText="Change"
+        cancelText="Cancel"
+        onIonChange={(e) => setChapter(e)}
+    >
+        {navState.chapters?.map((chapter) => {
+            return (
+                <IonSelectOption value={chapter} key={chapter}>
+                    {chapter}
```

Comment:
Could we add 'Ch ' before the chapter number here, just so it's easier to understand what this control is showing?